Doc/c-api/arg.rst

```diff
@@ -343,7 +343,7 @@ Other objects
    *items*.  Format units for sequences may be nested.
 
 It is possible to pass "long" integers (integers whose value exceeds the
-platform's :const:`LONG_MAX`) however no proper range checking is done --- the
+platform's :c:macro:`LONG_MAX`) however no proper range checking is done --- the
```

Comment:
```suggestion
platform's :c:macro:`!LONG_MAX`) however no proper range checking is done --- the
```

Reply:
I think that it may be better to solve this for standard C names globally.